src/main/java/processes/Ui.java

```diff
@@ -0,0 +1,130 @@
+package processes;
+
+
+import java.util.ArrayList;
+
+import tasks.Task;
+
+/**
+ * The task that deals with user inputs.
+ * Formats the input for use in other classes, such as the TaskList class.
+ */
+public class Ui {
+
+    /**
+     * Called when the programme just starts.
+     * Prints the welcome message onto the terminal to greet the user.
+     *
+     * @param name Name of the chatbot
+     * @return The welcome message to print in the GUI
+     */
+    public String showWelcomeMessage(String name) {
+        return "Hello! I'm " + name + "\n";
+    }
+
+    /**
+     * Called when the programme is terminated.
+     * Returns the goodbye message onto the terminal for the user.
+     *
+     * @return The goodbye message
+     */
+    public String showGoodbyeMessage() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+
+    /**
+     * Returns the list of tasks on the terminal for the user.
+     *
+     * @param taskList The current list of tasks
+     * @return The string containing the list of tasks
+     */
+    public String showTaskList(ArrayList<Task> taskList) {
+        if (taskList.size() > 0) {
+            StringBuilder res = new StringBuilder("Your current tasks are: \n");
+            for (int i = 0; i < taskList.size(); i++) {
+                res.append((i + 1) + "." + taskList.get(i).toString() + "\n");
+            }
+            return res.toString();
+        } else {
+            return "You currently have no tasks!";
+        }
+
+    }
+
+    /**
+     * Takes in a message and prints it onto the terminal, useful for showing exception messages.
+     *
+     * @param message The message that should be printed onto the terminal
+     * @return The message to print in the GUI
+     */
+    public String showMessage(String message) {
+        return message;
+    }
+
+    /**
+     * Takes in the task that was marked and returns the feedback.
+     * Informs the user that the task is successfully marked
+     *
+     * @param markedTask The task that was just marked.
+     * @return The success message
+     */
+    public String showMarked(Task markedTask) {
+        return "task\n  " + markedTask.toString() + "\nis marked!";
+    }
+
+    /**
+     * Takes in the task that was unmarked and returns the feedback.
+     * Informs the user that the task is successfully unmarked
+     *
+     * @param unmarkedTask The task that was just unmarked.
+     * @return The success message
+     */
+    public String showUnmarked(Task unmarkedTask) {
+        return "task\n  " + unmarkedTask.toString() + "\nis unMarked!";
+    }
```

Comment:
Could it be better to uncapitalize the 'M' in 'unmarked' for the returned `String`?
The method name `showUnmarked` implies 'unmarked' is being treated as a single word, however in the returned `String`, the capitalization is instead 'unMarked'.